main.go

```diff
@@ -151,6 +158,9 @@ func main() {
 				if argv.SQLNullStr {
 					g.WithSQLNullStr()
 				}
+				if templates := []string(argv.TemplateFileNames.Value()); len(templates) > 0 {
+					g.WithTemplates(templates...)
```

Comment:
We should probably include globbing in here as well, since it's supported at the filename level as well.